sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/README.md

```diff
@@ -1,36 +1,145 @@
 # Azure WebJobs Event Hubs client library for .NET
 
-TODO
+This extension provides functionality for accessing Azure Event Hubs in Azure Functions.
```

Comment:
Should we consider "from" instead of "in"?

```suggestion
This extension provides functionality for accessing Azure Event Hubs from an Azure Function.
```